### DISCUSSION TITLE
Samples - Adding ChangeTracking Capabilities to Database

### DISCUSSION THREAD

**Discussion by keithnicholson** [Azure/azure-functions-sql-extension#923] [General]
I'm looking at running the samples.  I noticed that the database scripts in the folders do not implement ChangeTracking at the database or table levels.  Are they hidden elsewhere?  If not, I can branch off and add them as I write them myself.  Then submit a PR for acceptance or denial.

**Comment by Charles-Gagnon** [comment]
Currently it's just hardcoded as part of our tests - the samples don't include setting that up.

https://github.com/Azure/azure-functions-sql-extension/blob/0ec2e69f2e0d5b52b113e51669b714ab893bf8d0/test/Common/TestUtils.cs#L120

Having a "General Setup" script to create the database and enable change tracking would definitely be useful if you wanted to look into that! And if you're really looking to make this nice having a [SQL Project](https://learn.microsoft.com/en-us/sql/azure-data-studio/extensions/sql-database-project-extension-getting-started?view=sql-server-ver16) that users could run to easily set up the DB could be useful!

**Reply by keithnicholson** [reply]
Working on the SQL project.  Would you want it at the root or in the samples folder?

**Reply by Charles-Gagnon** [reply]
Samples folder makes sense to me!
